examples/appSpec/build-and-test.sh

```diff
@@ -1,2 +1,2 @@
 #! /bin/bash
-hcdev package --output app.hcpkg --strip-meta && cargo test -- --nocapture
+hcdev package --output app.hcpkg --strip-meta; cargo test -- --nocapture
```

Comment:
Why this? no point running cargo test if your package build doesn't succeed is there?

Reply:
Oh yeah my mistake. I actually thought they were the other way around as there is a problem that the cargo test will run before it has finished generating the json. It fails in both cases